File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size() , i = 0;
        sort(nums.begin(),nums.end());
        for( ; i < n  ; i++){
            if(nums[i] != i)
                return i; 
        }
        return i;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length ;
        int sumN = (n*(n+1)) / 2 ;
        for(int i = 0 ; i < n ; i++)
            sumN -= nums[i] ;
        return sumN ;
    } 
}